pkg/check/check.go

```diff
@@ -37,15 +37,27 @@ const (
 	StateWarning State = "warn"
 )
 
+type Error struct {
+	Severity    State  `json:"severity"`
+	ShortErr    string `json:"short_error"`
+	Left        string `json:"left,omitempty"`
```

Comment:
use a meaningful name instead of left and right in error message?

Reply:
that's corresponding to `column length mismatch (1 vs 2)`. How about self and other?

Reply:
make sense